README.md

```diff
@@ -42,7 +42,7 @@ brew install hbase
 
 To compile, build and run the application use the following command:
 ```shell
-sbt run
+gradle runPlayBinary
 ```
 The default application port is 9000. To specify an alternative port use `-Dhttp.port=8080`.
```

Comment:
Any idea how to override the default port with gradle?  Setting the JVM property -Dhttp.port does not appear to work ...

Reply:
I have seen this [feature](https://docs.gradle.org/current/dsl/org.gradle.play.tasks.PlayRun.html) in the docs - I will make the change so its included.

Reply:
The change is:

``` groovy
tasks.withType(PlayRun) { 
    httpPort = 9010 
}
```